src/test/groovy/com/michalpu/PokeRest/pokemon/infrastructure/api/PokemonEndpointTest.groovy

```diff
@@ -26,8 +29,26 @@ class PokemonEndpointTest extends BaseIntegrationTest {
         then:
         with(pokemon) {
             id == 4
-            name.equalsIgnoreCase("charmander")
-            weight == 85
+            name.equalsIgnoreCase("charmander");
+            weight == 86
         }
     }
+
+
+    void 'should return types vulnerable to pokemons type'  () {
```

Comment:
in the future, this test class would grow, so it's better to stick to the smaller classes even in the tests. One for collecting Pokemon's info, another type for relations, etc..